*** chroot/etc/mozpluggerrc.orig	2009-04-09 20:25:57.000000000 +0900
--- chroot/etc/mozpluggerrc	2009-04-09 20:30:08.000000000 +0900
***************
*** 177,183 ****
  audio/x-mid:midi,mid:MIDI audio file
  audio/midi:midi,mid:MIDI audio file
  audio/x-midi:midi,mid:MIDI audio file
! 	controls noisy stream: timidity -Od "$file"
  	controls: playmidi "$file"
  
  audio/mod:mod:Soundracker audio Module
--- 177,183 ----
  audio/x-mid:midi,mid:MIDI audio file
  audio/midi:midi,mid:MIDI audio file
  audio/x-midi:midi,mid:MIDI audio file
! 	controls noisy stream: timidity -Os "$file"
  	controls: playmidi "$file"
  
  audio/mod:mod:Soundracker audio Module
